src/components/SwapWidget/SwapWidget.tsx

```diff
@@ -31,6 +31,12 @@ import Timer from "../../components/Timer/Timer";
 import Modal from "react-modal";
 import Card from "../Card/Card";
 import WalletProviderList from "../WalletProviderList/WalletProviderList";
+import TokenSelection from "../../components/TokenSelection/TokenSelection";
+
+export interface TokenSelectionModalState {
```

Comment:
Where are you using this interface?

Reply:
Had an idea to use an interface - but scratched it and forgot lol. thanks for pointing it out :D